### PR TITLE
NONPR-1501 Update service Play libraries to 2.6/2.7

### PR DESCRIPTION
**Pull request by greg-gajda**

### CODE REVIEW

File: app/controllers/SearchController.scala [hunk 1]
```diff
@@ -30,44 +29,45 @@ import controllers.FormMappings._
...
-                                 implicit val appConfig: AppConfig,
+class SearchController @Inject()(@Named("retrieval-actor") retrievalActor: ActorRef,
                                  val authConnector: AuthConnector,
                                  val nrsRetrievalConnector: NrsRetrievalConnector,
-                                 implicit val system: ActorSystem,
-                                 implicit val mat: Materializer,
-                                 val searchResultUtils: SearchResultUtils) extends FrontendController with I18nSupport with Stride {
+                                 val searchResultUtils: SearchResultUtils,
+                                 override val controllerComponents: MessagesControllerComponents,
+                                 val searchPage: search_page,
```
Comment (opetch) [inline app/controllers/SearchController.scala:48]: do these value names need to be in snake_case?

Comment (greg-gajda) [reply app/controllers/SearchController.scala:48]: It's data type generated by play for a page template and I didn't change any pages names

**Comment by craigtd** [comment]
There are a lot of compile time warnings. I have no idea which of them are new:

**Comment by craigtd** [comment]
Down to 10 warnings about deprecated methods which I guess I can live with.

**Pull request merged**